### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by bygui86** [arminc/k8s-platform-lcm#8]
Can you please describe how to install this tool?

**Comment by arminc** [comment]
I made a fix in 0.6.0 so you can run it locally without needing the template. 
The most simple way to run it is download version 0.6.0 and run `./lcm --local` It will use the config.yaml you provided and will search for the versions.

If you want to see the web UI then you will need the template folder next to the lcm binary. I am working on a Docker image ( issue #11 ) so it's already integrated.

**Comment by bygui86** [comment]
👍 thanks!

Once Docker image will be ready, it would be great to add a specific section to the README.md :)

**Comment by blacs30** [comment]
For testing lcm  I created a very simple Dockerfile. In the folder of the Dockerfile I've downloaded and placed the files from the templates and static folder as well as the config file.

```
FROM ubuntu:18.04

ENV VERSION 0.6.0

RUN apt update && apt install net-tools curl -y && curl -L https://github.com/arminc/k8s-platform-lcm/releases/download/$VERSION/lcm-$VERSION-linux -o /bin/lcm && chmod +x /bin/lcm && rm -rf /var/lib/apt/lists/*

COPY ./exampleConfig.yaml /
COPY ./templates /templates
COPY ./static /static

CMD /bin/lcm --server --config /exampleConfig.yaml
```

A simple clusterrole and clusterrolebinding for kubernetes could look this:

```
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: ClusterRole
metadata:
  labels:
    app: lcm
    release: lcm
  name: lcm
rules:
- apiGroups:
  - ''
  resources:
  - namespaces
  - pods
  verbs:
  - list
---
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: ClusterRoleBinding
metadata:
  labels:
    app: lcm
    release: lcm
  name: lcm
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: lcm
subjects:
- kind: ServiceAccount
  name: default
  namespace: kube-system
```

Seems like a promising tool, thanks @arminc .

**Comment by arminc** [comment]
Will be fixed by #12
